app/views/follow_requests/create.js.erb

```diff
@@ -0,0 +1,3 @@
+var existingButton = $("#<%= dom_id(@follow_request.sender)%>_<%= dom_id(@follow_request.recipient)%>_follow_unfollow");
```

Comment:
create and destroy do the same thing. consider making one file ("toggle") and reference format.js { toggle } in the controller's create and destroy methods.